components/admin_console/billing/payment_info.tsx

```diff
@@ -23,25 +23,24 @@ type Props = {
 
 const PaymentInfo: React.FC<Props> = () => {
     const dispatch = useDispatch<DispatchFunc>();
-    const customer = useSelector((state: GlobalState) => state.entities.cloud.customer);
 
-    const isCreditCardAboutToExpire = () => {
+    const isCardAboutToExpire = useSelector((state: GlobalState) => {
+        const {customer} = state.entities.cloud;
```

Comment:
Same here, why can't the customer be fetched from the state in the previous `useSlelector`?

Reply:
The problem didn't come from the selector to get the customer information, it was the local component state not being updated when the redux state was changed. I'm not 100% on why that was happening, I'm still fairly new to react hooks. 

I could separate back out the customer into its own selector, but figured it was unnecessary since we only need it in this scope.